Emoji.Wpf/Emoji.Wpf.csproj

```
@@ -1,9 +1,5 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props"
-          Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
-  <UsingTask TaskName="GZipTask" TaskFactory="CodeTaskFactory"
-             AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.v4.0.dll">
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
```

I don't believe the `WindowsDesktop` SDK is needed any more.

```suggestion
﻿<Project Sdk="Microsoft.NET.Sdk">
```

---

Just tried it but unfortunately it breaks net40 target, so had to revert.

---

@drewnoakes, I think this is the issue: https://github.com/dotnet/sdk/issues/15475

---

I made some tests the project does not build without it, seemingly because of `<UseWPF>true</UseWPF>`.

---

@samhocevar are these tests checked in? Everything builds for me if you have `Sdk="Microsoft.NET.Sdk.WindowsDesktop"`